ex3/tema3.js

```diff
@@ -0,0 +1,14 @@
+function letterCountNr(str, letter) 
+{
+ let letterCount = 0;
```

Comment:
```suggestion
    let letterCount = 0;
```
(optional) dupa fiecare acolada deschisa, se vrea un tab (4 spatii) de identare a codului